server/server.go

```diff
@@ -216,6 +217,13 @@ func main() {
 }
 
 func ping(w http.ResponseWriter, _ *http.Request) {
+	if err := store.Ping(); err != nil {
+		sentry.CaptureException(errors.Wrap(err, "database ping failed"))
+		w.WriteHeader(500)
+		_, _ = w.Write([]byte("database ping failed"))
```

Comment:
you can just use `.status(500)` I believe, the text you send back doesn't really matter.